### PR TITLE
Validation fix for empty SIB scenes.

### PR DESCRIPTION
**Pull request by rmitton**
The validator requires empty scenes to explicitly have NULL pointers.

### CODE REVIEW

File: code/SIBImporter.cpp [hunk 1]
```diff
@@ -860,9 +860,9 @@ void SIBImporter::InternReadFile(const std::string& pFile,
     pScene->mNumMaterials = sib.mtls.size();
     pScene->mNumMeshes = sib.meshes.size();
     pScene->mNumLights = sib.lights.size();
-    pScene->mMaterials = new aiMaterial* [pScene->mNumMaterials];
-    pScene->mMeshes = new aiMesh* [pScene->mNumMeshes];
-    pScene->mLights = new aiLight* [pScene->mNumLights];
+	pScene->mMaterials = pScene->mNumMaterials ? new aiMaterial*[pScene->mNumMaterials] : 0;
+	pScene->mMeshes = pScene->mNumMeshes ? new aiMesh*[pScene->mNumMeshes] : 0;
+	pScene->mLights = pScene->mNumLights ? new aiLight*[pScene->mNumLights] : 0;
```
Comment (kimkulling) [inline code/SIBImporter.cpp]: Coud you please use NULL istead of 0?

File: code/SIBImporter.cpp [hunk 2]
```diff
@@ -875,7 +875,7 @@ void SIBImporter::InternReadFile(const std::string& pFile,
     aiNode *root = new aiNode();
     root->mName.Set("<SIBRoot>");
     root->mNumChildren = sib.objs.size() + sib.lights.size();
-    root->mChildren = new aiNode* [root->mNumChildren];
+	root->mChildren = root->mNumChildren ? new aiNode*[root->mNumChildren] : 0;
```
Comment (kimkulling) [inline code/SIBImporter.cpp]: Same here!

File: code/SIBImporter.cpp [hunk 3]
```diff
@@ -889,7 +889,7 @@ void SIBImporter::InternReadFile(const std::string& pFile,
         node->mTransformation = obj.axis;
 
         node->mNumMeshes = obj.meshCount;
-        node->mMeshes = new unsigned[node->mNumMeshes];
+		node->mMeshes = node->mNumMeshes ? new unsigned[node->mNumMeshes] : 0;
```
Comment (kimkulling) [inline code/SIBImporter.cpp]: Same here!

**Comment by rmitton** [comment]
No problem, should all be fixed now.

**Comment by acgessler** [comment]
Looks good - thanks!

**Pull request merged**